### DISCUSSION TITLE
interstellar links send me to classlink

### DISCUSSION THREAD

**Discussion by replitidk** [UseInterstellar/Interstellar#823] [Q&A] [answered]
interstellar links have been sending me  to classlink im not sure if this is a feature and i have to press a button to get to interstellar or if this is my school having a clever work around.
![image](https://github.com/user-attachments/assets/5964af61-5057-4d65-8c39-0f91618e6b88)

**Comment by 0mhx** [comment, marked as answer]
it's a feature to hide it better

**Reply by replitidk** [reply]
ok thanks but another question how do I exactly access it
